src/gc/gcee.cpp

```diff
@@ -74,38 +74,33 @@ void GCHeap::UpdatePreGCCounters()
 
     GetPerfCounters().m_Security.timeRTchecks = 0;
     GetPerfCounters().m_Security.timeRTchecksBase = 1; // To avoid divide by zero
-
 #endif //ENABLE_PERF_COUNTERS
 
-#ifdef FEATURE_EVENT_TRACE
 #ifdef MULTIPLE_HEAPS
         //take the first heap....
     gc_mechanisms *pSettings = &gc_heap::g_heaps[0]->settings;
 #else
     gc_mechanisms *pSettings = &gc_heap::settings;
 #endif //MULTIPLE_HEAPS
 
-    ETW::GCLog::ETW_GC_INFO Info;
-
-    Info.GCStart.Count = (uint32_t)pSettings->gc_index;
-    Info.GCStart.Depth = (uint32_t)pSettings->condemned_generation;
-    Info.GCStart.Reason = (ETW::GCLog::ETW_GC_INFO::GC_REASON)((int)(pSettings->reason));
+    uint32_t count = (uint32_t)pSettings->gc_index;
+    uint32_t depth = (uint32_t)pSettings->condemned_generation;
+    uint32_t reason = (uint32_t)pSettings->reason;
 
-    Info.GCStart.Type = ETW::GCLog::ETW_GC_INFO::GC_NGC;
+    uint32_t type = /*ETW::GCLog::ETW_GC_INFO::GC_NGC*/ 0;
```

Comment:
These values are tied to the GC, but happen to live in the VM.  What do you think about moving them into the GC implementation so that they can be used directly?

Reply:
Will do! I'm running into trouble with shared types between the EE and GC so I'll see what I can separate.